pkg/bpf/bpffs_linux.go

```diff
@@ -185,6 +196,8 @@ func checkOrMountCustomLocation(bpfRoot string) error {
 		if err := mountFS(); err != nil {
 			return err
 		}
+
+		return nil
```

Comment:
Why return nil here now vs. before?

Reply:
It's to ensure that we print `log.Infof("Detected mounted BPF filesystem at %s", mapRoot)` only when the BPF FS was not mounted by the agent itself.

Reply:
Happy to restructure this if you have a better idea. The main motivation is to be able to grep for a particular message to proof that the BPF filesystem was already mounted.

Reply:
@tgraf makes sense, thanks!